app/main.py

```diff
@@ -1 +1,17 @@
-# write your code here
+import os
+
+
+def move_file(command: str) -> None:
+    split_command = command.split()
```

Comment:
Code Style Issue: The variable name `split_command` is not very descriptive. Consider renaming it to something more meaningful, like `command_parts`.